src/index.html

```diff
@@ -1,13 +1,60 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
-</html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img 
+        src="images/imac.jpeg" 
+        alt="APPLE A1419 iMac 27&#8243; Retina 5K Monoblock (MNED2UA/A)" 
+        class="card__img">
+  
+      <h2 class="card__description">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+  
+      <p class="card__product-code">
+        Product code: 195434
+      </p>
+  
+      <div class="card__rating">
+        <div class="card__stars stars stars--4">
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+        </div>
+  
+        <div class="card__reviews">
+          <p>Reviews: 5</p>
```

Comment:
```suggestion
        <p class="card__reviews">
          Reviews: 5
```
Can't you just use paragraph instead of div?